linux/powershell/Invoke-PreparePowerShell.ps1

```diff
@@ -17,25 +17,15 @@ $moduleList = @(
     "SHiPS",
     "AzurePSDrive",
     "MicrosoftPowerBIMgmt",
+    "Az",
     "Az.GuestConfiguration",
     "Microsoft.PowerShell.UnixCompleters"
 )
 
-if ($PSVersionTable.PSEdition -eq "Core")
-{
-    $moduleList += "AzureAD.Standard.Preview"
```

Comment:
Are we removing AAD module?

Reply:
If yes, we need to remove test case in PSinLinuxCloudShellImage.Tests.ps1:
It "Initialize AzureAD.Standard.Preview Module"

Reply:
We are not removing it from the image, but from the warmup script. We already changed the startup so we do not usually load this module whenever pwsh is started, because sometimes it takes a very long time. So this just means we don't load it during the 'prewarm' phase.